tests/infra/logging_app.py

```diff
@@ -145,7 +161,7 @@ def _verify_tx(
                         LOG.info(
                             f"Sleeping for {retry_after}s waiting for historical query processing..."
                         )
-                        timeout += retry_after
+                        timeout += int(retry_after * 0.8)
```

Comment:
0.8?

Reply:
The logic here is that we want to increase the overall loop timeout (which is 3 seconds by default) if we're verifying historical query. It used to be `timeout += retry_after`, which meant it would take many iterations of 3 seconds (30 iterations as the default timeout is 3 and we sleep 0.1 seconds in the loop anyway) to time out. `0.8` seems a nice compromise to go through this loop enough times without causing a ridiculous timeout if no historical entry can be found. I've added an inline comment to explain this.